src/main/resources/userEmailTemplate-feedbackSessionClosing.html

```diff
@@ -9,7 +9,7 @@
     <br><strong>To submit feedback for the above session, please go to this Web address: </strong>
     <a href="${submitUrl}">${submitUrl}</a>
     <br>*The above link is unique to you. Please do not share it with others.
-    <br>You may ignore this email if you have already submitted feedback.
+    <br><strong>You may ignore this email if you have already submitted feedback.</strong>
```

Comment:
Shall we use the same phrasing in both emails? `No action is required if you have already submitted responses for this session.`